mobile-app/app/screens/AppNavigator/screens/Auctions/screens/PlaceBidScreen.tsx

```diff
@@ -0,0 +1,313 @@
+  const {
+    bottomSheetRef,
+    containerRef,
+    dismissModal,
+    expandModal,
+    isModalDisplayed,
+    bottomSheetScreen,
+    setBottomSheetScreen
+   } = useBottomSheet()
+
+  const navigation = useNavigation<NavigationProp<AuctionsParamList>>()
+  const hasPendingJob = useSelector((state: RootState) => hasTxQueued(state.transactionQueue))
+  const hasPendingBroadcastJob = useSelector((state: RootState) => hasBroadcastQueued(state.ocean))
+  const blockCount = useSelector((state: RootState) => state.block.count) ?? 0
+  const { blocksRemaining } = useAuctionTime(vault.liquidationHeight, blockCount)
+
+  const [bidAmount, setBidAmount] = useState<string>('')
+
+  const onBidMinAmount = (val: string): void => {
+    setBidAmount(val)
+  }
+
+  const onPressFullDetails = (): void => {
+    setBottomSheetScreen([{
+      stackScreenName: 'Auctioned collaterals',
+      option: {
+        header: () => null,
+        headerBackTitleVisible: false
+      },
+      component: BottomSheetAuctionedCollateral({
+       collaterals: batch.collaterals,
+       headerLabel: translate('screens/PlaceBidScreen', 'Auctioned collaterals'),
+       onCloseButtonPress: dismissModal
+      })
+    }])
+    expandModal()
+  }
+
+  const onSubmit = async (): Promise<void> => {
+    if (hasPendingJob || hasPendingBroadcastJob) {
+      return
+    }
+
+    navigation.navigate('ConfirmPlaceBidScreen', {
+      batch,
+      bidAmount: new BigNumber(bidAmount),
+      estimatedFees,
+      totalAuctionValue: totalLoanAmountInUSD,
+      vault
+    })
+  }
+
+  const ownedTokenAmount = ownedToken === undefined ? '0' : ownedToken.amount
+  const isValidMinBid = bidAmount === '' || new BigNumber(bidAmount).gte(minNextBidInToken)
+  const hasSufficientFunds = new BigNumber(ownedTokenAmount).gte(minNextBidInToken)
+
+  return (
+    <View ref={containerRef} style={tailwind('h-full')}>
+      <ThemedScrollView
+        testID='place_bid_screen'
+        contentContainerStyle={tailwind('flex flex-col justify-between py-6 pb-8 px-4 h-full')}
+      >
+        <View>
+          <BidSummaryCard
+            displaySymbol={batch.loan.displaySymbol}
+            collateralDisplaySymbols={batch.collaterals.map(collateral => collateral.displaySymbol)}
+            blockCount={blockCount}
+            liquidationHeight={vault.liquidationHeight}
+            minNextBid={minNextBidInToken}
+            totalAuctionValue={totalLoanAmountInUSD}
+            onPressFullDetails={onPressFullDetails}
+          />
+
+          <WalletTextInput
+            autoCapitalize='none'
+            onChange={(e: NativeSyntheticEvent<TextInputChangeEventData>) => {
+              onBidMinAmount(e.nativeEvent.text)
+            }}
+            placeholder='0.00'
+            style={tailwind('flex-grow w-2/5')}
+            value={bidAmount}
+            displayClearButton={new BigNumber(bidAmount).gte('0.00')}
+            onClearButtonPress={() => onBidMinAmount('0.00')}
```

Comment:
Also let's use empty string here after `onClearButtonPress`, so that we will not force user to enter a decimal value.
Right now: enter some amount > press on clear button > input reset to 0.00 > focus in input again, the input is starting from 0.00, let's align with other input to not assign the `0.00`

Reply:
Updated